lambdas/question/src/main/java/uk/gov/di/ipv/cri/kbv/api/handler/QuestionHandler.java

```diff
@@ -134,6 +141,30 @@ public APIGatewayProxyResponseEvent handleRequest(
         return response;
     }
 
+    private void sendNoQuestionAuditEvent(QuestionsResponse questionsResponse) throws SqsException {
+        auditService.sendAuditEvent(
+                AuditEventType.THIRD_PARTY_REQUEST_ENDED,
+                createAuditEventContext(questionsResponse));
+    }
+
+    private void setAuthCode(KBVItem kbvItem) {
+        SessionItem sessionItem = sessionService.getSession(String.valueOf(kbvItem.getSessionId()));
+        sessionService.createAuthorizationCode(sessionItem);
+    }
+
+    private Map<String, Object> createAuditEventContext(QuestionsResponse questionsResponse) {
```

Comment:
I wondered if this could move into `questionResponse` so we could call it like `questionResponse.createAuditEventContext()` since this is also in the `answerHandler`

Reply:
let's do it in another PR